rest/route/status.go

```diff
@@ -85,13 +101,29 @@ func (h *recentTasksGetHandler) Run(ctx context.Context) gimlet.Responder {
 		return gimlet.NewJSONResponse(response)
 	}
 
-	models := make([]model.Model, 1)
+	if h.byDistro || h.byProject {
```

Comment:
seems like we handle these cases a few times. Does it make more sense to return early in other cases?

Reply:
Where else?